core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -89,6 +91,41 @@ object Invoker {
 
         val msgProvider = SpiLoader.get[MessagingProvider]()
         val producer = msgProvider.getProducer(config, ec)
+
+        val invokerOrdinal =
+          if (args.length == 0) {
+            val registrationUUID = UUID()
```

Comment:
If an invoker restarts on same host should it have a different UUID? Currently this will orphan messages queued for the ID just before it exited. I thought that the invoker on same host should have the same instance ID on restart. This implies that the ID is derived from the host (or cluster of hosts) somehow - we currently leverage the host IP (encoded as an int) for this purpose